lib/linter/haml.rb

```diff
@@ -3,7 +3,7 @@ class Haml < Base
     private
 
     def config_files
-      [".haml-lint.yml"]
+      [".haml-lint.yml"] + Linter::Rubocop::CONFIG_FILES
```

Comment:
Do you want to create the same constant here for consistency?

Reply:
I could? but I think I'd rather do it in a follow up.